plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -200,42 +177,44 @@ contract RootChain {
     function challengeExit(uint256 cUtxoPos, uint256 eUtxoPos, bytes txBytes, bytes proof, bytes sigs, bytes confirmationSig)
         public
     {
+        var txList = txBytes.toRLPItem().toList(11);
+        // Checks that spent input submit is the same as the one being exited
+        require(txList[0].toUint() + txList[1].toUint() + txList[2].toUint() == eUtxoPos);
```

Comment:
Should it be `require(txList[0].toUint() * 1000000000  + txList[1].toUint() * 10000 + txList[2].toUint() == eUtxoPos);`?